### ISSUE TITLE
Tab autocomplete not working on remote server

### ISSUE THREAD

**Issue by Mr-Arceni** [turikhay/passwordmanager-for-authme#8]
Autofill when generating and pasting existing passwords works in the local world, but not when logging into a remote server.
The `pwam` command still works though. Could this be related to some server modifications/settings?

**Comment by turikhay** [comment]
Yes, it's probably related. If the server is public, could you post the IP address here? If not, I'll have to ask you to post the log file, redacting anything that looks confidential.
